threat-miner/info.json

```diff
@@ -0,0 +1,278 @@
+                }
+            ],
+            "parameters": [
+                {
+                    "title": "File Hash",
+                    "type": "text",
+                    "name": "file_hash",
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "value": "e6ff1bf0821f00384cdd25efb9b1cc09",
+                    "placeholder": "e.g: e6ff1bf0821f00384cdd25efb9b1cc09",
+                    "description": "Retrieved the metadata for the specified file hash(md5, sha1, sha256).",
+                    "tooltip": "Retrieved the metadata for the specified file hash(md5, sha1, sha256)."
+                },
+                {
+					"title": "Query Type",
+					"type": "select",
+					"name": "query_type",
+					"required": true,
+					"visible": true,
+					"editable": true,
+                    "description": "Select the type of query that is how you want to perform samples operation.",
+                    "tooltip": "Select the type of query that is how you want to perform samples operation.",
+					"options": [
+						"Metadata",
+						"HTTP Traffic",
+						"Hosts (domains and IPs)",
+                        "Mutants",
+						"AV detections",
+                        "Report tagging"
+					]
+				}
+            ]
+        },
+		{
+            "title": "Get Import Hash Details",
+            "operation": "get_import_hash_details",
+            "category": "investigation",
+            "description": "Reports samples,report tagging used in malware analysis to identify malware binaries that belong to the same family.",
+            "annotation": "import_hash_details",
+            "output_schema": [
+                {
+                    "status_code": "",
+                    "status_message": "",
+                    "results": []
+                }
+            ],
+            "parameters": [
+                {
+                    "title": "Imphash",
+                    "type": "text",
+                    "name": "imphash",
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "value": "1f4f257947c1b713ca7f9bc25f914039",
+                    "placeholder": "e.g: 1f4f257947c1b713ca7f9bc25f914039",
+                    "description": "It requires a hash value to retrieve the malware analysis report.",
+                    "tooltip": "It requires a hash value to retrieve the malware analysis report."
+                },
+                {
+					"title": "Query Type",
+					"type": "select",
+					"name": "query_type",
+					"required": true,
+					"visible": true,
+					"editable": true,
+                    "description": "Select the type of query that is how you want to perform imphash operation.",
+                    "tooltip": "Select the type of query that is how you want to perform imphash operation.",
+					"options": [
+						"Samples"
```

Comment:
Do we need to add Report tagging option as well?

Reply:
The Report tagging API is not working. So, removed the select query type field. By default samples query types will be executed.